x-pack/plugins/security_solution/public/timelines/components/flyout/header/kpis.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { EuiStat, EuiFlexItem, EuiFlexGroup } from '@elastic/eui';
+import { TimelineKpiStrategyResponse } from '../../../../../common/search_strategy';
+import { getEmptyValue } from '../../../../common/components/empty_value';
+import * as i18n from './translations';
+
+export const TimelineKPIs = React.memo(
+  ({ kpis, isLoading }: { kpis: TimelineKpiStrategyResponse | null; isLoading: boolean }) => {
+    return (
+      <EuiFlexGroup wrap data-test-subj="siem-timeline-kpis">
+        <EuiFlexItem>
+          <EuiStat
+            data-test-subj="siem-timeline-process-kpi"
+            title={kpis === null ? getEmptyValue() : kpis.processCount}
```

Comment:
might just be able to cache the empty value in `useMemo(() => getEmptyValue());` and then call it in all of these. Or just make it functional `getTitle(actualVal) { return kpis === null ? cachedEmptyValue : actualVal; }`

Reply:
I am just wondering if that 's really need it because we are only dealing with a prop type of a string here and react should cache the props the right way through reference because it is a string.